config/dev/nginx.conf.template

```diff
@@ -176,7 +176,7 @@ server {
       return 204;
     }
     proxy_set_header Host $http_host;
-    proxy_pass http://${MY_DOCKER_HOST}:2103/$1;
+    proxy_pass http://${MY_DOCKER_HOST}:2103/$1$is_args$args;
```

Comment:
Add a information on migration document to say that user has to update their nginx.conf and what to modify